### PR TITLE
GrafanaUI: Change chevron directions for open/closed state

### LINKED ISSUE #47273: Expandable Sidemenu: Chevron direction on sections (Dashboards, Alerts etc.). When a section is collapsed, chevron should point down, when expanded, should point up

### PR DESCRIPTION
**Pull request by joshhunt**
**What this PR does / why we need it**:

For new nav, we want to change the chevron direction so when a section is collapsed, chevron should point down, when expanded, should point up.

Decided to change this in the component, and change it everywhere. 🔥  Controversial 🔥 

This impacts:

 - [Alerting ChannelSettings](https://github.dev/grafana/grafana/blob/25e04d77be3fecba00902b5312f011c95589f901/public/app/features/alerting/components/ChannelSettings.tsx#L23)
 - [Alerting NotificationSettings](https://github.dev/grafana/grafana/blob/25e04d77be3fecba00902b5312f011c95589f901/public/app/features/alerting/components/NotificationSettings.tsx#L11)
 - [AnnotationSettings](https://github.dev/grafana/grafana/blob/25e04d77be3fecba00902b5312f011c95589f901/public/app/features/dashboard/components/AnnotationSettings/AnnotationSettingsEdit.tsx#L99)
 - and a few places in [DashboardSettings](https://github.dev/grafana/grafana/blob/25e04d77be3fecba00902b5312f011c95589f901/public/app/features/dashboard/components/DashboardSettings/GeneralSettings.tsx#L144)

In a separate PR, I want to remove `CollapsableSection` from most of DashboardSettings and just replace it with a FieldSet. "Graph tooltip" doesn't need to be collapsible.

**Which issue(s) this PR fixes**:

Fixes #47273

**Special notes for your reviewer**:

### CODE REVIEW

**Review by ashharrison90 — APPROVED** [review]
i'm all for this, but curious to hear others opinions 🙂

**Comment by zoltanbedi** [comment]
This was recently changed by https://github.com/grafana/grafana/pull/46950 @andresmgot

**Comment by joshhunt** [comment]
Good spot @zoltanbedi!

Seems like before @andresmgot's change, Collapse and CollapsibleSection had different directions, and he aligned them. So _IF_ (big IF) we were to go ahead with this, I would want to revert his change in this as well.

**Comment by joshhunt** [comment]
`<Collapse />` is used in a lot more places than `<CollapsibleSection />` so I'm less willing to make that change. 

I think I will err on the side of not doing this and closing this PR.

**Comment by andresmgot** [comment]
Since I got pinged I will give my opinion :) 

Since the CollapsibleSection has the icon on the right side, I think it's fine for it to be up / down (it doesn't make sense for that to be pointing to the right).

Since the Collapsible has the icon on the left side, it does make sense to have the icon pointing to the right.

Look at these examples of Material UI, which does the same that I mention: https://mui.com/components/accordion/

So in other words, my +1 to this change (without changing the Collapsible component).

**Comment by andresmgot** [comment]
Since I got pinged I will give my opinion :) 

Since the CollapsibleSection has the icon on the right side, I think it's fine for it to be up / down (it doesn't make sense for that to be pointing to the right).

Since the Collapsible has the icon on the left side, it does make sense to have the icon pointing to the right.

Look at these examples of Material UI, which does the same that I mention: https://mui.com/components/accordion/

So in other words, my +1 to this change (without changing the Collapsible component).
